gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/NetworkService.java

```diff
@@ -52,6 +55,8 @@ public class NetworkService {
 
     private final MinioAdapter minioAdapter;
 
+    private final Map<String, String> subNetworkIdByCountry = new HashMap<>();
```

Comment:
I find it risky to use an instance attribute on a singleton: nobody can tell what will happen if there are parallel treatments using NetworkService...